bga_database/static/js/employer.js

```diff
@@ -182,6 +183,7 @@ var Department = {
     ChartHelper.make_payroll_expenditure_chart(result.payroll_expenditure);
 
     Employer.updateSearchLink('#employee-search-link', year);
+    Employer.updateSearchLink('#employee-download-link', year);
```

Comment:
actually, it's pretty confusing that this method is called updateSearchLink. the method seems like it could be just renamed to updateLink, right?

Reply:
yeah that's confusing. changed here: ac4d4d6